frontend/components/EmptyState/_styles.scss

```
@@ -0,0 +1,215 @@
+      font-size: $small; // 16px per Figma
+      font-weight: $bold;
+      line-height: 1.5;
+      color: $core-fleet-black;
+      margin: 0;
+    }
+  }
+
+  &__info,
+  &__additional-info {
+    @include help-text;
+    line-height: 1.5;
```

Stylelint (17.9.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)